server/schedulers/evict_slow_store.go

```diff
@@ -111,8 +111,9 @@ func (s *evictSlowStoreScheduler) EncodeConfig() ([]byte, error) {
 }
 
 func (s *evictSlowStoreScheduler) Prepare(cluster schedule.Cluster) error {
-	if len(s.conf.EvictedStores) != 0 {
-		return s.prepareEvictLeader(cluster)
+	evictStore := s.conf.evictStore()
```

Comment:
The original way can reduce the number of function calls.

Reply:
The origin way visits the arguments of `s.conf` directly which is not a good way, using the function of `s.conf` instead would be clear, that is way we need to define the struct of `conf` II think